IOSAccessAssessment/Services/ChangesetService.swift

```diff
@@ -0,0 +1,124 @@
+//
+//  ChangesetService.swift
+//  IOSAccessAssessment
+//
+//  Created by Mariana Piz on 11.11.2024.
+//
+
+import Foundation
+
+struct NodeData {
+    let latitude: Double
+    let longitude: Double
+}
+
+class ChangesetService {
+    
+    private enum Constants {
+        static let baseUrl = "https://osm.workspaces-stage.sidewalks.washington.edu/api/0.6"
+        static let workspaceId = "168"
+    }
+    
+    static let shared = ChangesetService()
```

Comment:
I like the idea of using singletons, and imo we should have used something like that for the SharedImageData as well (along with adding it to a global store, instead of passing it to every view)